lix/source-code/server/src/git-proxy.ts

```diff
@@ -44,7 +44,7 @@ export async function proxy(request: Request, response: Response, next: NextFunc
 	}
 	try {
 		// remove the proxy path from the url
-		request.url = request.url.split(privateEnv.PUBLIC_GIT_PROXY_PATH)[1]!
+		request.url = request.url.split("/git-proxy/")[1]!
 
```

Comment:
Removing the null assertion and minimize the risk for bugs. 


```suggestion
		const withoutProxyPath = request.url.split("/git-proxy/")[1]
		if (withoutProxyPath === undefined) {
			throw new Error("Proxy path not found.")
		}
		request.url = withoutProxyPath
```